### DISCUSSION TITLE
Help! why don't work my code when i try to catch the method "saludo" in the prototype

### DISCUSSION THREAD

**Discussion by VictorRColussi** [codewars/codewars.com#2610] [General]
Hi!!! i need capt the function "saludo" in the prototype "Personaje" in mi objet "jefeOrco" but i don't understand becouse don´t work my call if i heredity jefeOrco of Orco and of Personaje. i wan't use Class. in this project. i want capt saludo in "jefeOrco of de Personaje.prototype.saludo()

Sorry about my english.

```

function Personaje(nombre,materia,coin){
  
  this.nombre = nombre,
  this.origen = materia,
  this.puntaje = coin

}

Personaje.prototype.saludo = function(){
  
    {console.log("hola")}  
    //console.log("Hola soy "+this.nombre +" mi nivel es "+this.puntaje+" , juega conmigo");
  }


function Orco(nombre, materia, coin, habilidad, escudo) {
  
  this.callclass = Personaje;
  this.callclass(nombre,materia,coin);
  
  this.poder = habilidad;
  this.proteccion = escudo 
  
  
}


const jefeOrco = new Orco("Rawco","Tierra",10,["Pluma de roca","Terremoto fatal","Manada Sangrienta"],"Burbuja de tierra")


Orco.prototype.despedida = function(){
  
    {console.log("chau")}  
    //console.log("Hola soy "+this.nombre +" mi nivel es "+this.puntaje+" , juega conmigo");
  }


//jefeOrco.saludo()

jefeOrco

//jefeOrco.saludo()


```

**Comment by JohanWiltink** [comment]
Please fix your markup. Start and end a code block with three backticks on a line by themself.

Also, whatever you have against `class` syntax, it probably makes it easier for people to help you if you use it, because it makes your code more concise.

I have difficulty reading your English ( Google translate? ), but I can't hold that against you. I would like to try to help you, but please make it as easy as possible for me. Fix that markup!

**Comment by JohanWiltink** [comment]
Your constructor `Orco` is apparently intended to extend your class `Personaje`. It doesn't. `callclass` is not a magic spell, it's an ordinary property.

You want a class `Personaje`, and you want a class `Orco` to `extend Personaje`.

That explanation should be enough. If it is beyond your grasp, you should be doing simpler things for now.

**Comment by VictorRColussi** [comment]
thank's a lot!!!! yes i understand that you say. 

💯
